public/js/reader.js

```diff
@@ -112,6 +133,25 @@ Reader.initializeSettings = function () {
     localStorage.removeItem("containerwidth");
 };
 
+Reader.initInfiniteScrollView = function () {
+    $("#i1").addClass("infinite-scroll");
+    $("#Map").remove();
+    $(".reader-image").first().attr("src", Reader.pages[0]);
+
+    Reader.pages.slice(1).forEach((source) => {
+        const img = new Image();
+        img.src = source;
```

Comment:
You mean lazy loading? It's something I'd like to add it in a future iteration. We'll have to use an exernal module like http://jquery.eisbehr.de/lazy/, because baking something ad-hoc is a nightmare (speaking from experience, the part dealing with scrolling position forces you to write spaghetti code). I didn't add it this time around since I didn't even plan on adding an infinite scroller at first, but then I broke the existing one, so I had to.

When I (eventually) do this I also plan to add things like per-image reload buttons, like the ones exhentai has, to offset network issues.

Reply:
Welp, that module looks perfect for the job indeed! 👏 Don't have much else to say on this point, it was just an observation for now.